os2rollekatalog_integration/os2rollekatalog_integration.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2019, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import requests
+
+from os2mo_tools import mo_api
+
+
```

Comment:
These thing should be specified either in the configuration or as command line parameters.

Reply:
Indeed; we should probably create a library for that given that this is at least our third package requiring configuration.